src/rpg_objects/Game_Temp.js

```diff
@@ -55,3 +56,16 @@ Game_Temp.prototype.destinationX = function() {
 Game_Temp.prototype.destinationY = function() {
     return this._destinationY;
 };
+
+Game_Temp.prototype.tileMapSortRequested = function() {
+  return this._tilemapSortRequested === true;
```

Comment:
`return this._tilemapSortRequested` seems enough. On the other hand, there is no coding rule in this project yet, so I'd like to ask core maintainers of this project.